staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -58,7 +58,7 @@ type TypeMeta struct {
 // ListMeta describes metadata that synthetic resources must have, including lists and
 // various status objects. A resource may have only one of {ObjectMeta, ListMeta}.
 type ListMeta struct {
-	// selfLink is DEPRECATED read-only field that is no longer populated by the system.
+	// selfLink is a legacy read-only field that is no longer populated by the system.
```

Comment:
I agree about not shouting in the godoc, but putting `Deprecated: ...` at the start of a line by itself to trigger deprecation warnings on use seems useful to me. should we do that?

Reply:
That makes sense to me - updated.

Reply:
Mostly a separate concern, but it might be handy to have an extra string match for fields that are both deprecated (please don't use this) and actually-unused (even if you try it doesn't work). It might not come through for OpenAPI but SIG Docs could consume that kind of data, maybe.

If there's any sort of existing convention for marking no-longer-served fields, let's definitely use that here.